backend/tournesol/views/contributor_recommendations.py

```diff
@@ -38,18 +38,16 @@ def annotate_with_total_score(self, queryset, request, poll: Poll, user):
             total_score=Sum(
                 F("contributorvideoratings__criteria_scores__score") * criteria_weight,
             ),
-        )
+        ).filter(total_score__isnull=False)
```

Comment:
a similar filter is missing in `PollsRecommendationsViews` in polls.py, line 215